qadence/__init__.py

```diff
@@ -1,10 +1,39 @@
 from __future__ import annotations
 
+import logging
+import logging.config
+import os
 from importlib import import_module
+from pathlib import Path
 
+import yaml
 from torch import cdouble, set_default_dtype
 from torch import float64 as torchfloat64
 
+DEFAULT_FLOAT_DTYPE = torchfloat64
+DEFAULT_COMPLEX_DTYPE = cdouble
+set_default_dtype(DEFAULT_FLOAT_DTYPE)
```

Comment:
default dtype should surely be set before including files that might depend on it? Just in case

Reply:
jep